### PR TITLE
Translate python/documentation.md

### PR DESCRIPTION
**Pull request by sjlee2016**
I translated the documentation section for python-sdk 
If there is any error, please let me know! 

@ianychoi  @yeongseon

### CODE REVIEW

File: docs/python/documentation.md [hunk 1]
```diff
@@ -1,41 +1,42 @@
 ---
-title: "Python Guidelines: Documentation"
+title: "파이썬 가이드 라인 : 문서"
```
Comment (ianychoi) [inline docs/python/documentation.md:2]: "가이드 라인" -> "가이드라인" (there is no space)

File: docs/python/documentation.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
+* `API reference` - 코드의 the docstrings에서 생성되며 docs.microsoft.com에 게시됩니다. 
+* `Code snippets` - 라이브러리에서 식별한 시나리오에 대한 단일(원자) 연산을 보여주는 간단한 코드 예입니다. 이는 README, docstrings, and Quickstart 에 사용됩니다. 
+* `Quickstart` - README 와 유사하지만 확장되는 docs.microsoft.com 관련 문서입니다. 일반적으로 서비스 콘텐츠 개발자가 작성합니다.  
+* `Conceptual` - Quickstarts, 튜토리얼, 사용 방법 가이드 및 기타 컨텐츠와 같은 긴 형식의 설명서입니다. 일반적으로 서비스 콘텐츠 개발자가 작성합니다. 
 
-{% include requirement/MUST id="python-docs-content-dev" %} include your service's content developer in the adparch review for your library. To find the content developer you should work with, check with your team's Program Manager.
+{% include requirement/MUST id="python-docs-content-dev" %} 서비스의 콘텐츠 개발자를 라이브러리에 대한 임시 보관소 리뷰어로 포함하시오. 함께 작업해야 할 콘텐츠 개발자를 찾으려면 팀의 프로그램 관리자에게 문의하십시오.
 
-{% include requirement/MUST id="python-docs-contributor-guide" %} follow the [Azure SDK Contributors Guide]. (MICROSOFT INTERNAL)
+{% include requirement/MUST id="python-docs-contributor-guide" %} [Azure SDK Contributors 가이드라인] 을 따르시오. (MICROSOFT INTERNAL)
```
Comment (ianychoi) [inline docs/python/documentation.md:20]: Suggestion: "가이드라인] 을" -> "안내서]를"

File: docs/python/documentation.md [hunk 3]
```diff
@@ -1,41 +1,42 @@
...
 ## Docstrings
 
-{% include requirement/MUST id="python-docstrings-pydocs" %} follow the [documentation guidelines](http://aka.ms/pydocs) unless explicitly overridden in this document.
+{% include requirement/MUST id="python-docstrings-pydocs" %} 이 문서에서 명시적으로 재정의되지 않은 경우 [문서 작성 가이드라인](http://aka.ms/pydocs) 을 따르세요.
 
-{% include requirement/MUST id="python-docstrings-all" %} provide docstrings for all public modules, types, and methods.
+{% include requirement/MUST id="python-docstrings-all" %} 모든 공용 모듈, 유형 및 메서드에 대한 docstrings를 제공합니다.
 
-{% include requirement/MUST id="python-docstrings-kwargs" %} document any `**kwargs` directly consumed by a method and add a ref link to [core options](https://aka.ms/azsdk/python/options) to provide introduction for shared options. You may refer to the signature of a called method if the `**kwargs` are passed through.
+{% include requirement/MUST id="python-docstrings-kwargs" %} 메소드에서 직접적으로 사용되는 모든  `**kwargs` 를 문서화하고 [core options](https://aka.ms/azsdk/python/options)에 참조 링크를 추가하여 공유 옵션에 대한 소개를 제공합니다. `**kwargs` 가 전달되면,호출된 메소드의 서명을 참조 할 수 있습니다.
```
Comment (ianychoi) [inline docs/python/documentation.md:37]: "메서드" is a preferred translation for method - see https://www.microsoft.com/ko-kr/language/Search?&searchTerm=method&langID=457&Source=true&productid=0

**Review by ianychoi — CHANGES_REQUESTED** [review]
Good and requesting with some comments

**Comment by sjlee2016** [comment]
I made a few improvements based on the comments I received. I changed all "메소드" to "메서드" and used "안내서" Instead of "가이드 라인". Thank you for your comments!

**Pull request merged**